packages/nimble-components/src/select/index.ts

```diff
@@ -961,13 +1078,9 @@ export class Select
     }
 
     private initializeOpenState(): void {
-        if (!this.open) {
-            this.ariaExpanded = 'false';
-            this.ariaControls = '';
-            return;
-        }
-
+        this.openActiveIndex = this.selectedIndex;
         this.committedSelectedOption = this.options[this.selectedIndex];
+        this.setActiveOption(this.openActiveIndex ?? this.selectedIndex);
```

Comment:
You don't need the `?? this.selectedIndex` on this line, right? You just initialized `this.openActiveIndex = this.selectedIndex` two lines up.